lib/appraisal/utils.rb

```diff
@@ -7,6 +7,13 @@ def self.support_parallel_installation?
       Gem::Version.create(Bundler::VERSION) >= Gem::Version.create("1.4.0.pre.1")
     end
 
+    def self.support_git_local_installation?
+      Gem::Version.create(Bundler::VERSION) > Gem::Version.create("2.4.22")
+    end
```

Comment:
There may be other places we'd like to integrate this check.  So far it is only used in specs.